smtp.go

```diff
@@ -33,26 +34,35 @@ func (v *Verifier) CheckSMTP(domain, username string) (*SMTP, error) {
 	}
 
 	var ret SMTP
+	var err error
+	email := fmt.Sprintf("%s@%s", username, domain)
 
 	// Dial any SMTP server that will accept a connection
-	client, err := newSMTPClient(domain, v.proxyURI)
+	client, mx, err := newSMTPClient(domain, v.proxyURI)
```

Comment:
Is it necessary to check the domain obtained by mx resolution, instead of directly determining the domain name

Reply:
Take an example, `xxx@aftership.com` belongs to gmail and its mx.Host is `aspmx.l.google.com.`. If we determine by domain, `@aftership.com` will not be recognized as gmail.